static/app/views/settings/account/accountClose.tsx

```diff
@@ -165,18 +166,20 @@ function AccountClose() {
 
           {organizations?.map(({organization, singleOwner}) => (
             <PanelItem key={organization.slug}>
-              <label>
-                <input
-                  style={{marginRight: 6}}
-                  type="checkbox"
-                  value={organization.slug}
-                  onChange={evt => handleChange(organization, singleOwner, evt)}
-                  name="organizations"
-                  checked={orgsToRemove.has(organization.slug)}
-                  disabled={singleOwner}
-                />
-                {organization.slug}
-              </label>
+              <Checkbox
+                css={{
+                  marginRight: 6,
+                  display: 'flex',
+                  alignItems: 'center',
+                }}
+                name="organizations"
+                checked={orgsToRemove.has(organization.slug)}
+                disabled={singleOwner}
+                value={organization.slug}
+                onChange={evt => handleChange(organization, singleOwner, evt)}
+                size="sm"
+              />
+              {organization.slug}
```

Comment:
i think this would mean that you can no longer click on the org slug to check the box? Wrapping in a label element might fix this

Reply:
ah good catch, just fixed